docs/annexes/annex-2/annex-2-high-level-requirements.md

```diff
@@ -1210,7 +1210,7 @@ B.  Restore Requirements 
 
 The most relevant scenario for this functionality is when a user would
 like to migrate from one Wallet solution to a different Wallet solution.
-Another scenario would be when the Wallet solution provider ceases to
+Another scenario would be when the Wallet solution Provider ceases to
```

Comment:
@pinamiranda Why should "provider" be capitalized in a case like this? In this text flow, "provider" is a common noun, part of the sentence, and does not refer to any specific or special provider. 

Typically, we capitalize words when they are not part of the natural sentence flow, and we need to emphasize that they refer to something previously defined elsewhere. 

### Examples on capitalization:

- **Common Noun**: When the noun is not specific and is used in a general sense.
  - "... the details are defined in the relevant article." (article is not specific)

- **Proper Noun**: When the noun refers to a specific entity previously defined.
  - "... more details about this topic are provided in **Article** 5.1." (Article 5.1 is a specific section)

### Application to our Case

In our context, "provider" would typically be a common noun and not capitalized unless it refers to a specific entity previously defined or agreed upon to be treated as a proper noun within the document or context. Here are some examples of when "provider" might be capitalized:

1. **Common Noun**:
   - "The **provider** will be responsible for delivering the services." (general reference to any provider)

2. **Proper Noun**:
   - "The **Provider** has specific responsibilities outlined in Section 4." (if "Provider" is defined as a specific entity in a contract or legal document)